inlong-sort/sort-connectors/pulsar/src/main/java/org/apache/inlong/sort/pulsar/table/PulsarDynamicTableFactory.java

```diff
@@ -284,7 +287,8 @@ public DynamicTableSource createDynamicTableSource(Context context) {
                 serviceUrl,
                 adminUrl,
                 properties,
-                startupOptions);
+                startupOptions,
+            inLongMetric);
```

Comment:
Below is an identical variable called "inlongMetric", can this be the same as it?

Reply:
I guess it's nessary to create a Constant for connectors.